common/country_types/hfe_country_types.txt

```diff
@@ -209,13 +249,25 @@ hfe_drone = {
 					base = 33
 				}
 			}
+			ancient_drone_station = {
+				min = 1
+				max = 14
+			}
+			ancient_mining_drone = {
+				min = 10
+				max = 200
+			}
 		}
 	}
 	faction = {
 		hostile = yes
+		hostile_when_attacked = yes
```

Comment:
make no sense, always hostile

Reply:
They are always hostile, but they also turn hostile when attacked. The two commands are different, so I made sure to program both.

Reply:
Country types very unstable and may be ruined in any patch in an unexpected way.
I see no sense with overloading country types with excess flags.